src/types/quiz.types.ts

```diff
@@ -0,0 +1,51 @@
+import {
+	ADD__ANSWER,
+	CLOSE__MODAL,
+	OPEN__MODAL,
+	SET__PLAYER__NAME,
+	SET__QUESTIONS,
+} from "../constants";
+
+export type Question = {
+	_id: string;
+	category: string;
+	correctAnswer: string;
+	options: string[];
```

Comment:
`Options` shouldn't be a string array, it should be another type with 
```javascript
{
 id: string,
text: string,
}
```

Reply:
done